### PR TITLE
Adding seminar page and first two talks!

### PR DESCRIPTION
**Pull request by crazy4pi314**
I have added a page now for the seminar series, as well as the first two talks. The bios and abstracts are verbatim from the speaker form responses.

I tried to match the design here to the rest of the site as much as possible, let me know what you think!

### CODE REVIEW

File: blog.html [hunk 1]
```diff
@@ -114,6 +114,7 @@
                         <a href="grants.html">grants</a>
                         <a href="ideas.html">ideas</a>
                         <a href="mitiq.html">mitiq</a>
+                        <a href="seminars.html">seminars</a>
```
Comment (willzeng) [inline blog.html:117]: Could we call this "talks" instead of seminars? It looks a bit less intimidating in the menu bar

File: seminars.html [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+                        <a href="https://en.wikipedia.org/wiki/Unitarity_(physics)" target="_blank">unitary</a></b>.</p>
+                        <button id="mobile-menu-icon" class="hamburger hamburger--squeeze" type="button">
+                            <span class="hamburger-box">
+                              <span class="hamburger-inner"></span>
+                            </span>
+                        </button>
+                    </div>
+
+                    <p class="subtitle leading-arrow"><a name="seminars">Quantum Software Seminars</a></p>
+                    The Unitary Fund is pleased to host a online talk series that features open source quantum software projects. All seminars will be hosted at <a href="https://www.twitch.tv/unitaryfund">twitch.tv/unitaryfund</a>, and if you want to chat with the speakers or the community more generally, check out the <a href="https://discord.gg/JqVGmpkP96">Unitary Fund Discord</a>.
```
Comment (willzeng) [inline seminars.html:54]:
```suggestion
                    The Unitary Fund hosts an online talk series about open source quantum software. Talks are hosted at <a href="https://www.twitch.tv/unitaryfund">twitch.tv/unitaryfund</a>, and if you want to chat with the speakers or the community more generally, check out the <a href="https://discord.gg/JqVGmpkP96">Unitary Fund Discord</a>.
```

File: seminars.html [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+                            </p>
+                            <p>
+                                <strong>Alex McCaskey</strong> is a research scientist in the Computer Science and Mathematics Division at Oak Ridge National Laboratory. He serves as the Software Lead for the Quantum Computing Institute at ORNL and is the Project Lead for the XACC quantum framework and the QCOR quantum-classical C++ compiler. He received his Masters in Physics from Virginia Tech and BS degrees in Physics and Mathematics from the University of Tennessee.  
+                            </p>
+                        </li><br>
+                    </ul>
```
Comment (willzeng) [inline seminars.html:83]: Currently this amount of text is a bit overwhelming and could get very long quickly as we add talks. There's a few ways to deal with this, but perhaps it is easiest just to have titles and speaker names that hyperlink to the project and the speaker?

Am open to other suggestions

**Comment by willzeng** [comment]
Thanks @crazy4pi314! Added some quick comments

File: seminars.html (hunk 3 above)
Comment (crazy4pi314) [reply seminars.html:83]: Basically I need a place to put their full abstract and bio... I'm gonna try some collapsible details elements and see how that works

File: blog.html (hunk 1 above)
Comment (crazy4pi314) [reply blog.html:117]: I've updated that everywhere then to talks

File: style.css [hunk 4]
```diff
@@ -199,6 +199,35 @@ input{
...
+color: royalblue;
+} */
+
+summary::marker, summary::-webkit-details-marker {
+  display: none;
+}
+
+summary:after {
+  /* background: red; */
+  /* border-radius: 5px; */
```
Comment (willzeng) [inline style.css:216]:
```suggestion
```

File: style.css [hunk 5]
```diff
@@ -199,6 +199,35 @@ input{
...
 }
 
+/* details[open] summary::marker {
+color: red;
+}
+
+details:not([open]) summary::marker {
+color: royalblue;
+} */
+
```
Comment (willzeng) [inline style.css:209]:
```suggestion
```

File: style.css [hunk 6]
```diff
@@ -199,6 +199,35 @@ input{
...
+summary:after {
+  /* background: red; */
+  /* border-radius: 5px; */
+  content: "+";
+  color: #000;
+  float: left;
+  font-size: 1.5em;
+  font-weight: bold;
+  margin: -5px 10px 0 0;
```
Comment (willzeng) [inline style.css:222]:
```suggestion
  margin: -10px 10px 0 0;
```

Suggesting this adjustment as it helped alignment. Went from:
image
to
image

**Review by willzeng — CHANGES_REQUESTED** [review]
Nifty. The collapsing works well (and just with css!)

I made some very small suggestions for some left in commented lines and a small one on alignment, but after that should be good to go @crazy4pi314

**Review by willzeng — APPROVED** [review]
LGTM!

**Pull request merged**